include/devicetree/zephyr.h

```diff
@@ -36,12 +36,24 @@
  *        a label property, that property's value. Undefined otherwise.
  */
 #define DT_CHOSEN_ZEPHYR_ENTROPY_LABEL ""
+
+/**
+ * @def DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL
+ *
+ * @brief If there is a chosen node zephyr,flash-controller property which has
+ *        a label property, that property's value. Undefined otherwise.
+ */
+#define DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL ""
 #endif /* DT_DOXYGEN */
 
 #if DT_NODE_HAS_PROP(DT_CHOSEN(zephyr_entropy), label)
 #define DT_CHOSEN_ZEPHYR_ENTROPY_LABEL DT_LABEL(DT_CHOSEN(zephyr_entropy))
 #endif
 
+#if DT_NODE_HAS_PROP(DT_CHOSEN(zephyr_flash_ctlr), label)
```

Comment:
missed this one

Reply:
thanks for catching them.  My search replace didn't look for the underscore version.